response_operations_ui/templates/reporting-unit-survey.html

```diff
@@ -67,9 +67,9 @@ <h3 id="COLLECTION_EXERCISES" class="ons-u-fs-m">Collection exercises</h3>
                         {% set status_class = 'ons-status--error' %}
                     {% endif %}
 
-                    {% if ce.responseStatus in ['Not started', 'In progress'] and isReportingUnitPermission  %}
+                    {% if ce.responseStatus in ['Not started', 'In progress','Completed', 'Completed by phone', 'No longer required'] and isReportingUnitPermission  %}
                         {% set survey_link = '&nbsp;   <a href="' + url_for("case_bp.get_response_statuses", ru_ref=ru.sampleUnitRef, survey=survey.shortName, period=ce.exerciseRef) + '">Change</a>' %}
-                    {% elif ce.responseStatus in ['Completed', 'Completed by phone', 'No longer required'] and isReportingUnitPermission  %}
+                    {% elif isReportingUnitPermission  %}
```

Comment:
I'm unsure about this since I added the permission check in the next page as well so this might be worth just being an else and the `view` link

Reply:
I decided to just make this an if else and show the view to all users as that is how it is used across the rest of the site. It just made more sense to me